packages/beacon-node/test/unit/monitoring/service.test.ts

```diff
@@ -213,26 +216,30 @@ describe("monitoring / service", () => {
       assertError({message: new TimeoutError("request").message});
     });
 
-    it("should abort pending requests if monitoring service is closed", (done) => {
-      const endpoint = `${baseUrl}${remoteServiceRoutes.pending}`;
-      service = new MonitoringService("beacon", {endpoint, collectSystemStats: false}, {register, logger});
+    it("should abort pending requests if monitoring service is closed", () =>
+      new Promise<void>((done, error) => {
```

Comment:
don't we have to await this promise here? I'd assume otherwise this test case just finishes right away without waiting

Reply:
It's returning the promise, so eventually gonna be awaited for.

Reply:
This patterns feels kinda wrong to me but I guess can keep it 🤷